app/views/layouts/_user_menu.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="col-md-3 mu-tab-body mu-user-menu">
+  <div>
+    <div class="mu-user-menu-section">
+      <div class="mu-user-menu-header">
+        <%= t(:my_account) %>
+      </div>
+      <div class="mu-user-menu-item">
+        <%= profile_user_menu_link %>
+      </div>
+    </div>
+    <div class="mu-user-menu-section">
+      <div class="mu-user-menu-header">
+        <%= t(:messages) %>
+      </div>
```

Comment:
I'd like to find a better name than `messages` with this section, knowing that one of the items is `messages` itself (`discussions` is the other one).

Reply:
I think this was addressed in the call we had (if we end up using nameless separators)

Reply:
In the end we chose to have a single title